### ISSUE TITLE
metrics-server 0.5.1 version uses a imgpkgBundle that does not exists

### ISSUE THREAD

**Issue by beltran-rubo** [vmware-tanzu/community-edition#2742]
## Bug Report

metrics-server 0.5.1 uses "projects.registry.vmware.com/tce/metrics-server@sha256:393e7aa6c89f55229e9559f82165c3a51fe408c4c7dd4b72409844513b8caf83" taht does not exist into the registry.

## Expected Behavior

The image for that package should exists into the registry.

## Screenshots or additional information and context

```
    Updated At:          2021-12-17T12:50:03Z
  Friendly Description:  Reconcile failed: Fetching resources: Error (see .status.usefulErrorMessage for details)
  Observed Generation:   1
  Useful Error Message:  vendir: Error: Syncing directory '0':
  Syncing directory '.' with imgpkgBundle contents:
    Imgpkg: exit status 1 (stderr: imgpkg: Error: Checking if image is bundle:
  Fetching image:
    GET https://projects.registry.vmware.com/v2/tce/metrics-server/manifests/sha256:393e7aa6c89f55229e9559f82165c3a51fe408c4c7dd4b72409844513b8caf83:
      NOT_FOUND: artifact tce/metrics-server@sha256:393e7aa6c89f55229e9559f82165c3a51fe408c4c7dd4b72409844513b8caf83 not found
```

**Comment by joshrosso** [comment]
@seemiller can you take a look at diagnosing this at your earliest convenience?

**Comment by joshrosso** [comment]
Just remembered that `metrics-server` comes in the core repo, so this shouldn't impact users at runtime of official releases. Likely and artifact of pulling a package off main or from some point in source control.

cc @seemiller would be great to verify this theory. For now, moving into `v0.11.0`

**Comment by seemiller** [comment]
This issue was fixed with #2758
